### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by Feresey**

### CODE REVIEW

File: aur.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ type response struct {
 type By int
 
 const (
-	Name By = iota
+	Name By = iota + 1
```
Comment (mikkeloscar) [inline aur.go]: What's the benefit of changing this? If you need it for the test can't you test for something that is > the max?

Comment (Feresey) [reply aur.go]: The default value is zero. This means that sometimes, due to an error, zero may get into the program and main program will work incorrectly.

Comment (Feresey) [reply aur.go]: In short it is "foolproof"

Comment (mikkeloscar) [reply aur.go]: OK, I guess it's not a problem to change as it doesn't change the API (if it's being used in the right way).

**Pull request merged**